overlord/servicestate/service_control_test.go

```diff
@@ -292,11 +294,12 @@ func (s *serviceControlSuite) TestControlStartInstruction(c *C) {
 
 	inst := &servicestate.Instruction{
 		Action: "start",
-		Names:  []string{"foo"},
+		Names:  nil,
```

Comment:
what I meant is not to set the field at all which is equivalent to this, doesn't matter too much

Reply:
Let me change it then, indeed it makes sense to remove the assignation altogether.